test/e2e/apimachinery/resource_quota.go

```diff
@@ -111,7 +111,7 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		framework.ExpectNoError(err)
 
 		ginkgo.By("Not allowing a LoadBalancer Service with NodePort to be created that exceeds remaining quota")
-		loadbalancer := newTestServiceForQuota("test-service-lb", v1.ServiceTypeLoadBalancer, false)
+		loadbalancer := newTestServiceForQuota("test-service-lb", v1.ServiceTypeLoadBalancer, true)
```

Comment:
What's the reason for this change?

Reply:
This seems not relevant, whether we create NP or not for this service, it should fail. Will change it back.

Reply:
Revisited the case and sorted it out. Actually this is a bug in the test. The creation request was rejected because the `allocateLoadBalancerNodePorts` was turned off by default (thus each LB type service will allocate a node port and exceed quota and failed). Now as the feature is enabled by default, the creation will success if we don't allocate a node port to the service. We need to set it to true here so that the node port quota will exceed. Btw I've added more tests for this `allocateLoadBalancerNodePorts` feature in a separate conformance test so it's not interfering with the conformance tests written since 1.16.

Reply:
We should have a case where we pass true and it fails and another where we pass false and it does not fail.